PayloadPkg/OsLoader/OsLoader.c

```diff
@@ -700,6 +755,11 @@ PayloadMain (
   }
 #endif
 
+  // Check if there is a pre-OS checker that needs to be executed
+  if (PldBase != NULL) {
+    LoadPreOsChecker (PldBase);
```

Comment:
Can we let LoadPreOsChecker() return the entry ?
mPreOsCheckerEntry  = LoadPreOsChecker (PldBase);